enacts/calc.py

```diff
@@ -736,7 +792,7 @@ def seasonal_onset_date(
     # + seasonal_onset_date.onset_delta
     return seasonal_onset_date
 
-def seasonal_cess_date(
+def seasonal_cess_date_from_sm(
```

Comment:
Ugh. If we have to make two different versions of lots of different functions, that takes the air out of my argument for splitting cess_date_from_sm and cess_date_from_rain. Is this as far as it goes or is it likely to propagate further?

Reply:
This was mostly to have the Maprooms work. If our work in PR #372 turn out successfully, those named seasonal_ xxx functions will become obsolete and we will apply our seasonal operators to whatever functions.